src/lib/components/AppBar.react.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import React, { Component } from 'react';
+import { default as MuiAppBar } from '@material-ui/core/AppBar';
+
+type Props = {
+  /** Appbar ID */
+  id: string,
+  /** Children to render inside of the Appbar */
+  children?: Node,
+  /** The classes to be applied to this component. This keys in this object must be valid CSS rule
+   * names, and the values must be strings for the classnames to be assigned to each rule name
+   * Valid rule names are:
+   *   root
+   *   docked
+   *   paper
+   *   paperAnchorLeft
+   *   paperAnchorRight
+   *   paperAnchorTop
+   *   paperAnchorBottom
+   *   paperAnchorDockedLeft
+   *   paperAnchorDockedTop
+   *   paperAnchorDockedRight
+   *   paperAnchorDockedBottom
+   *   modal
+   * */
+  classes?: {
+    root?: string,
+    positionFixed?: string,
+    positionAbsolute?: string,
+    positionSticky?: string,
+    positionStatic?: string,
+    positionRelative?: string,
+    colorDefault?: string,
+    colorPrimary?: string,
+    colorSecondary?: string,
+    colorInherit?: string,
+    colorTransparent?: string,
+  },
+  /** CSS class name of the root element */
+  className?: string,
+  /** The color of the component. It supports those theme colors that make sense for this component. */
+  color?: string,
```

Comment:
Just as with `position`, would you enumerate the allowed possibilities for this prop?